GOVERNANCE.md

```diff
@@ -1,43 +1,35 @@
+# Commonhaus Foundation Project Governance
 
-- **Concerns and Actions**: Members can voice concerns or propose actions by emailing the [`council` mailing list][CONTACTS.yaml] (TODO: or initiating a GitHub discussion, etc.).
+Welcome to the Commonhaus Foundation (CF), a nonprofit dedicated to nurturing open-source projects through collaborative innovation.
 
-See [CF Council][cfc] for more details.
+## Governance Overview
 
-## CF Membership and Roles
+The Foundation operates in accordance with the [CF Bylaws][bylaws], which outline our operational principles and practices.
 
-- **Projects**: Open-source projects that are donated to CF. Project Leaders submit an application for their project to join the Commonhaus Foundation.
-- **Project Leaders and Code Owners**: Key contributors who drive CF projects. They hold admin and/or write permissions in project repositories. Project leaders are CF members and are eligible for election to the CFC.
-- **General Members**: CF Membership is open to all who support our mission. Active members, contributing in various ways, can gain voting rights and participate in elections.
+### CF Council (CFC)
 
-See [Membership and Roles][membership] for more details.
+The  [CFC][] oversees strategic and operational decisions. To engage with the council, email the [`council` mailing list][CONTACTS.yaml] or initiate a GitHub discussion.
 
-## CF Advisory Board
+### Extended Governance Committee (EGC)
 
-The **CF Advisory Board** comprises representatives from supporting organizations. They offer strategic guidance to the CFC.
+The [EGC][] is a joint body formed from members of the CFC and a representative for each foundation projet. Itplays a crucial role in ensuring diverse project perspectives are included in foundation-wide decisions. 
 
-See [CF Advisory Board (bylaws)][cfab] for more details.
+### Membership
 
-## Decision Making and Voting
+[CF Membership][membership] is open to individuals who support our mission. Active members gain voting rights and participate in elections.
 
-CF uses the Lazy Consensus decision-making model. When seeking consensus, members should express objections or dissent as early as possible to ensure there is time to discuss and address objections. See [Decision making (bylaws)][cf-decision-making] for more details.
+### Advisory Board
 
-For CFC elections, CF adopts a preference-based voting system facilitated by [Elekto](https://elekto.dev/).
+Organizations providing financial support to the CF appoint representatives to the [CF Advisory Board][cfab], which provides strategic guidance to the CFC.
 
-Every member, regardless of their role or contribution level, has an equal voice and is entitled to one vote in all decisions.
+### Decision Making
 
-## Record Keeping
+Decisions at CF follow the Lazy Consensus model, ensuring every member has a say. For details, see [Decision Making][cf-decision-making].
+### Record Keeping
```

Comment:
Do we have linting on the docs? We need a blank line before `### Record Keeping`

Reply:
I usually have it in place in the editor. I'll look at a linter when we work on next big chunk.